packages/components/src/AtlantisContext/AtlantisContext.tsx

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { createContext, useContext } from "react";
+
+export interface AtlantisContextProps {
+  /**
+   * The date format Atlantis components would use
+   */
+  readonly dateFormat: string;
+
+  /**
+   * The time format Atlantis components would use
+   */
+  readonly timeFormat: string;
+
+  /**
+   * Time zone used in converting the date and time formats
+   */
+  readonly timeZone: string;
+
+  /**
+   * Determines how the component behaves when the app is online or offline.
+   * This is set to `true` by default. If your usage doesn't require a different
+   * experience, you can leave this as-is.
+   */
+  readonly isOnline: boolean;
```

Comment:
React native specific so this can be removed. `onLogError` also isn't needed right now